server/testutils/factories/referral.ts

```diff
@@ -35,10 +36,39 @@ class ReferralFactory extends Factory<Referral> {
   }
 }
 
-export const randomStatus = (availableStatuses?: Array<ReferralStatus>) =>
-  faker.helpers.arrayElement(
-    availableStatuses || ['awaiting_assessment', 'assessment_started', 'referral_started', 'referral_submitted'],
-  ) as ReferralStatus
+const randomStatus = (availableStatuses?: Array<ReferralStatus>) =>
+  faker.helpers.arrayElement(availableStatuses || referralStatuses)
+
+const statusDescriptionAndColour = (status: ReferralStatus): Pick<Referral, 'statusColour' | 'statusDescription'> => {
```

Comment:
This is purely a helper for testing purposes and doesn't necessarily need to be accurate to what the API is returning at the moment.